File: src/me/riseremi/utils/NameGenerator.java
package me.riseremi.utils;

import java.util.Random;

/**
 *
 * @author riseremi <riseremi at icloud.com>
 */
public class NameGenerator {

    private static final Random rnd = new Random();
    private static final String[] names = {"File", "Edit", "View", "Run", "Debug",
        "Tools", "Window", "Help"};
    private static final String[] surnames = {"Navigate", "Source", "Refactor",
        "Profile", "Team", "Clean", "Build", "Save"};

    public static String getName() {
        return names[rnd.nextInt(names.length)] + " " + surnames[rnd.nextInt(surnames.length)];
    }

}

File: src/main/java/me/riseremi/utils/NameGenerator.kt
package me.riseremi.utils

import java.util.*

/**
 * @author riseremi <riseremi at icloud.com>
</riseremi> */
object NameGenerator {
    private val rnd = Random()
    private val names = arrayOf("Jack", "John", "Ken", "Dan", "Sophie", "Mary", "Kate", "Jane")
    private val surnames = arrayOf("White", "Stone", "Locke", "Wheeler", "Groves", "Brent", "Build", "Lincoln")

    fun getName() = names[rnd.nextInt(names.size)] + " " + surnames[rnd.nextInt(surnames.size)]
}
